provision-contest/ansible/presclient.yml

```diff
@@ -0,0 +1,30 @@
+---
+# This playbook installs the presentation client
+
+- name: Setup presentation client
+  hosts: presclient
+  vars:
+    host_type: presclient
+    GRAPHICAL: true
+  become: true
+  handlers:
+    - include: handlers.yml
+  roles:
+    - role: base_packages
+      tags: base_packages
+    - role: icpc_fixes
+      tags: icpc_fixes
+      when: ICPC_IMAGE
```

Comment:
```suggestion
```

Reply:
I prefer to keep this for if we ever do need this.